specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-07-30-preview/examples/DiskMigrations/Get.json

```diff
@@ -0,0 +1,52 @@
+{
+  "parameters": {
+    "subscriptionId": "04666444-56f0-4d4f-afc5-dbd6b134b084",
+    "location": "redmond",
+    "migrationId": "ec9f79cf-e4d7-4d37-926a-e69c2198be8a",
+    "api-version": "2018-07-30-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/04666444-56f0-4d4f-afc5-dbd6b134b084/providers/Microsoft.Compute.Admin/locations/redmond/diskmigrations/ec9f79cf-e4d7-4d37-926a-e69c2198be8a",
+        "name": "redmond/ec9f79cf-e4d7-4d37-926a-e69c2198be8a",
+        "type": "Microsoft.Compute.Admin/locations/diskmigrations",
+        "location": "redmond",
+        "properties": {
+          "migrationId": "ec9f79cf-e4d7-4d37-926a-e69c2198be8a",
+          "status": "Running",
+          "subtasks": [
+            {
+              "migrationSubTaskId": "726dff1c-e765-44c6-b16d-f66f2c28e730",
+              "properties": {
+                "migrationSubtaskStatus": "Succeeded",
+                "startTime": "2018-05-17T08:26:54.57146Z",
+                "endTime": "2018-05-17T08:27:35.0559255Z",
+                "targetShare": "\\\\SU1FileServer.mydomain.com\\SU1_ObjStore_1",
+                "sourceShare": "\\\\SU1FileServer.mydomain.com\\SU1_ObjStore_4",
+                "targetDiskStateForMigration": "Unattached",
+                "diskId": "423e4eb4-f791-4d13-ad5b-4d415733b0d6"
+              }
+            },
+            {
+              "migrationSubTaskId": "d6ab496c-4a76-474f-9ee2-9031053f06e8",
+              "properties": {
+                "migrationSubtaskStatus": "Running",
+                "startTime": "2018-05-17T08:27:35.0871773Z",
+                "endTime": "0001-01-01T08:00:00Z",
+                "targetShare": "\\\\SU1FileServer.mydomain.com\\SU1_ObjStore_1",
```

Comment:
All these should ideally be resource IDs (consistent with FRP, HRP etc.)

Reply:
We need to integrate with FRP to fetch the resource ID. currently diskrp doesn't do the integration. It is great to have. I will open a work item to track what we can do.